test/Error/error_cause_with_aggregate_error.js

```diff
@@ -0,0 +1,123 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Copyright (c) 2021 ChakraCore Project Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");

```

Comment:
Any reason this couldn't be kept in with the rest of the error_cause coverage?

Reply:
Because the arguments of aggregate errors. We need an errors array.

Reply:
Could wrap it like this: `function bar (...args) {return AggregateError([], ...args)}`

Would work for all cases other than the prototype one which you could do with a condition or something. I'd like to avoid this duplication if we can.

Reply:
And there's some access like Constructor.name......

Reply:
Please can we at least stick it into the same file?

Reply:
Sure.